src/app/controllers/admin/sessions_controller.rb

```diff
@@ -46,6 +46,12 @@ def create
     end
   end
 
+  def destroy
+    @session.destroy!
+    flash[:notice] = "Session has been deleted"
+    redirect_to '/admin/sessions'
```

Comment:
Consider changing this to use the router helper, e.g. something like:

`redirect_to admin_sessions_path()`

I don't have the app running, so I can't check the exact name, but you should see it listed in the output of `rails routes`